src/Tizen.NUI/src/public/Window/WindowEvent.cs

```diff
@@ -2021,10 +2173,24 @@ public event EventHandler<AccessibilityHighlightEventArgs> AccessibilityHighligh
                 {
                     using WindowAccessibilityHighlightEvent signal = new WindowAccessibilityHighlightEvent(Interop.WindowAccessibilityHighlightSignal.GetSignal(SwigCPtr), false);
                     signal.Ensure()?.Disconnect(AccessibilityHighlightEventCallback);
-                    AccessibilityHighlightEventCallback = null;
+                    ReleaseSafeCallback(ref AccessibilityHighlightEventCallback);
                 }
             }
         }
+
+        void CreateSafeCallback<T>(T method, out T safeCallback) where T : Delegate
```

Comment:
To maintain consistency with NUI coding rule, could you add `internal` or `private` to these functions here?

Reply:
Thank you for your comment. I will make another commit to add `private` keyword because this patch is to sync with [this](https://github.com/Samsung/TizenFX/pull/6501)  patch in `DevelNUI` branch so it would not be good to have any changes with it.

Reply:
Oh Okay!